src/main/java/it/gov/pagopa/tkm/ms/consentmanager/controller/impl/ConsentControllerImpl.java

```diff
@@ -22,4 +23,10 @@ public ConsentResponse postConsent(String taxCode, String clientId, Consent cons
         taxCode = taxCode.toUpperCase();
         return consentManagerService.postConsent(taxCode, clientId, consent);
     }
+
+    @Override
+    public GetConsentResponse getConsent(String taxCode, String hpan, String[] services) {
+        return consentManagerService.getConsent(taxCode, hpan, services);
```

Comment:
Manca l'upper sul consent and il lower sul hpan. Magari sarebbe utile creare una libreria/classe